libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/GranularPermissions/GranularPermissionItem.tsx

```diff
@@ -1,38 +1,80 @@
-// (C) 2022 GoodData Corporation
+// (C) 2022-2023 GoodData Corporation
+
+import React, { useCallback } from "react";
+import { useIntl } from "react-intl";
 
-import React from "react";
 import cx from "classnames";
 
-import { GranteeItem, IGranularPermissionTypeItem } from "../types";
+import { GranteeItem, IGranularGrantee, IGranularPermissionTypeItem } from "../types";
+
+import { Bubble, BubbleHoverTrigger } from "../../../../Bubble";
+import { IAlignPoint } from "../../../../typings/positioning";
+
+const alignPoints: IAlignPoint[] = [{ align: "cr cl" }];
 
 interface IGranularPermissionItemProps {
+    grantee: IGranularGrantee;
     permission: IGranularPermissionTypeItem;
+    selectedPermission: string;
     toggleDropdown: () => void;
     onChange: (grantee: GranteeItem) => void;
+    handleSetSelectedPermission: (permission: string) => void;
 }
 
-export const GranularPermissionItem: React.FC<IGranularPermissionItemProps> = ({
+const DropdownItem: React.FC<IGranularPermissionItemProps> = ({
```

Comment:
Rename it something more descriptive? Is it GranularPermissionsSelectItems ???  Or this items are generic and can be/ will be applied in DropDown in general?